ietf/meeting/models.py

```diff
@@ -1296,6 +1296,16 @@ def onsite_tool_url(self):
             return url.format(session=self)
         return None
 
+    def session_recording_url(self):
+        if (
+            self.meeting.type.slug == "ietf"
+            and self.has_onsite_tool
+            and (url := getattr(settings, "MEETECHO_SESSION_RECORDING_URL", ""))
```

Comment:
We've been avoiding the walrus to date to make it easier for newcomers to the codebase to contribute.
Not using it here would make the condition easier to read/debug anyhow